torch/ao/pruning/_experimental/pruner/README.md

```diff
@@ -1,93 +1,263 @@
-# Intro
+# Structured Pruning
+
+## Intro / Motivation
+
+**pruning** is the process of removing weights / connections in a neural network in order to make it run faster.
+
+There are two types of pruning that we support: **structured pruning** and **unstructured pruing**
+
+* **Unstructured pruning** refers to pruning any arbritary weight
+* **Structured pruning** refers to pruning by removing entire rows / columns of a weight.
```

Comment:
should be "of a weight tensor" i think it also should be indicated that these changes can propagate onwards